EosPayload/drivers/science_driver.py

```diff
@@ -4,7 +4,10 @@
 from EosPayload.lib.base_drivers.driver_base import DriverBase
 
 from adafruit_pm25.i2c import PM25_I2C
-from adafruit_blinka.microcontroller.am335x import pin
+try:
+    from adafruit_blinka.microcontroller.am335x import pin
+except RuntimeError:
```

Comment:
Would `except ImportError` be a bit more specific here than `RuntimeError`, if it works?

Reply:
honestly I just ran the program and whatever exception/error that occurred is what I put in the except clause.  In this case it died on a RuntimeError, nothing more specific